src/webui/api/torrentscontroller.cpp

```diff
@@ -1154,3 +1154,46 @@ void TorrentsController::tagsAction()
         result << tag;
     setResult(result);
 }
+
+void TorrentsController::renameFileAction()
+{
+    checkParams({"hash", "id", "name"});
```

Comment:
@Piccirello `checkParams` was renamed to `requireParams` 9b5df92078fbf6f7e7b8a78d432cac140d670751

Reply:
Thanks @thalieht